.travis.yml

```diff
@@ -29,19 +29,18 @@ os:
   - osx
   - windows
 julia:
-  - 1.0  # LTS
-  - 1.4  # Latest release
+  - 1.4  # Latest stable release
```

Comment:
I'd like to keep supporting the Long Term Stable release. I think a way that can be done here is to use Artifacts on 1.3+ and use the direct download for earlier versions

Reply:
I can't use Julia 1.3 here because of https://github.com/JuliaLang/Pkg.jl/issues/1912.
And the <1.4 compatibility should be done by having `if` on Julia version with direct download inside of `build`?

Reply:
That is what I'm thinking should be done

Reply:
It would be useful if the direct download is also done via BinaryProvider in `<v1.4`. That'll make things a little more consistent IMO. One example is at: https://github.com/invenia/LibPQ.jl/blob/master/deps/build.jl (you'll have to ignore the platform specific stuff since that's not relevant here)

Reply:
I'm not sure switching BinaryProvider has any benefits in this case besides being more consistent with other packages. One scenario to consider here is that although TimeZones specifies a version of tzdata to use with each release the package also supports using other versions of tzdata with a TimeZones release.

I think for this specific PR we should keep things straight forward an just add artifact support for Julia 1.4+ and leave the direct download as it was for now.

Reply:
Added running julia on 1.0, 1.3, 1.4, 1.5 and nightly, passing everywhere so it should be ok now, I hope.